src/Cake.Hosts/CakeHostsAliases.cs

```diff
@@ -37,6 +39,33 @@ public static bool HostsRecordExists(this ICakeContext context, String ipAddress
             return result;
         }
 
+        /// <summary>
+        /// Lista all records in hosts file
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <returns>If contains records, return all records</returns>
+        [CakeMethodAlias]
+        public static IEnumerable<HostsFile> GetAllRecords(this ICakeContext context)
+        {
+            var cakeHosts = GetCakeHosts(context);
+            var result = cakeHosts.GetAllRecords();
+
+            if (result.Any())
+            {
+                context.Log.Write(Verbosity.Normal, LogLevel.Information, "List all records in hosts file");
+                foreach (var item in result)
+                {
+                    context.Log.Write(Verbosity.Normal, LogLevel.Information, $"IP {item.Ip} with Hostname {item.Hostname}");
```

Comment:
I feel that should rather be a `Debug` level log output. 
However for `Normal` log output I'd say `$"Hosts file contains {count} records"`